clients/gtest/CMakeLists.txt

```diff
@@ -87,6 +96,10 @@ target_link_libraries( hipsolver-test PRIVATE
   Threads::Threads
   roc::hipsolver
 )
+if( hipblas_FOUND )
+  target_link_libraries( hipsolver-test PRIVATE roc::hipblas )
+  target_sources( hipsolver-test PRIVATE hipblas_include1_gtest.cpp hipblas_include2_gtest.cpp )
+endif( )
```

Comment:
Rather than checking `hipblas_FOUND`, could we add an option that the user needs to explicitly set? We can set it to `OFF` by default to minimize dependencies required for third-parties to build and test hipSOLVER, but pass the flag to enable it in the .jenkins groovy files (much like is done with `WERROR` on the rocSOLVER CI). This will prevent it from being enabled by mistake just because hipBLAS happened to be available on the build machine.

Reply:
Just wanted to highlight this comment, since I think it is the most important comment. I probably should have focused my comments on this topic, but brevity is not my strength.

Reply:
Good idea. I was thinking of adding a build option for my upcoming sparse PR as well, so this change should make things more consistent.